scopes/dependencies/dependency-resolver/dependency-linker.ts

```diff
@@ -619,6 +619,7 @@ function resolveModuleDirFromFile(resolvedModulePath: string, moduleId: string):
   }
 
   const [start, end] = resolvedModulePath.split('@');
+  if (!end) return path.basename(resolvedModulePath);
   const versionStr = head(end.split('/'));
```

Comment:
This might be not the best solution but it does fix a crash when the path doesn't contain '@'. I am open to suggestions.